app/models/post_presenter.rb

```diff
@@ -0,0 +1,6 @@
+module PostPresenter
+
+  def format_time
+    return "posted at #{self.created_at.strftime('%H:%M')} on #{self.created_at.strftime('%d/%m/%Y')}"
+  end
+end
```

Comment:
Looks great! I'd probably put it into 'app/presenters/post_presenter.rb'.

A class is more commonly used:

```ruby
class PostPresenter
  def initialize(post)
    @post = post
  end
  
  def timestamp
    post.created_at.strftimeblablablabl
  end
end

```